src/thing/thing.ts

```diff
@@ -245,31 +245,39 @@ function cloneLitStructs<Dataset extends SolidDataset>(
   return freshDataset as Dataset;
 }
 
-interface CreateThingLocalOptions {
+/** Pass these options to [[createThing]] to initialise a new [[Thing]] whose URL will be determined when it is saved. */
+export type CreateThingLocalOptions = {
   /**
    * The name that should be used for this [[Thing]] when constructing its URL.
    *
    * If not provided, a random one will be generated.
    */
   name?: string;
-}
-interface CreateThingPersistedOptions {
+};
+/** Pass these options to [[createThing]] to initialise a new [[Thing]] for which you already know at which URL it will be located. */
+export type CreateThingPersistedOptions = {
   /**
    * The URL of the newly created [[Thing]].
    */
   url: UrlString;
-}
+};
+/** The options you pass to [[createThing]]; either you give it a URL right away, or it gets determined when saving the Thing. See [[CreateThingPersistedOptions]] for the former, and [[CreateThingLocalOptions]] for the latter. */
```

Comment:
To avoid former/latter construct with the previous sentence (which cause eyeballs to shift back and forth form current and previous sentences), maybe

```recommendation
/** The options you pass to [[createThing]].
 * - To specify the URL for the initialised Thing, pass [[CreateThingPersistedOptions]].
 * - To have the URL determined during the save, pass [[CreateThingLocalOptions]].
 */
```